msticpy/data/storage/azure_blob_storage.py

```diff
@@ -244,7 +244,7 @@ def get_sas_token(
         key = self.abs_client.get_user_delegation_key(start, end)  # type: ignore
         abs_name = self.abs_client.account_name  # type: ignore
         sast = generate_blob_sas(
-            abs_name,
+            abs_name,  # type: ignore
```

Comment:
I can have a look to implement proper typing in that file so that the type ignore is not required. Would that help?

Reply:
Not sure how widely used this feature is. Maybe something for the future